src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List App</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+</head>
+<body>
+    <div class="todo-list">
+        <h3>Today's To Do</h3>
+
+        <div class="add-todo">
+            <input type="text" name="" placeholder="Add to your list...." class="input-task" required/>
+            <i class="fa fa-arrow-left"></i>
```

Comment:
![Screenshot from 2023-03-30 04-45-53](https://user-images.githubusercontent.com/93039359/228692770-d73ab1bb-71de-4a68-a98c-968e4676961f.png)

**[OPTIONAL]**

- [ ] It would be great if you put the arrow at the right corner to follow the design guidelines of this project :+1: